drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/test/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImplTest.java

```diff
@@ -102,6 +102,19 @@ public void remoteCallBackSortedIgnoreCase(){
                      dropDownListCaptor.getValue().stream().map(KieAssetsDropdownItem::getValue).collect(Collectors.toList()));
     }
 
+    @Test
+    public void getRemoteCallBackFilterTargetDirectory() {
+        RemoteCallback<List<String>> remoteCallBack = scenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImplSpy.getRemoteCallback(assetConsumer);
+        remoteCallBack.callback(Arrays.asList("/test/project/target/classes/file1", "path/B", "target/file1", "a", "C:\\test\\project\\target\\classes\\file1"));
```

Comment:
May we add here "/test/usertarget/targetGroup.dmn" so to test that userpackage ending with "target/" would not ve filtered?